src/components/options/PanelOptions.tsx

```diff
@@ -35,6 +35,7 @@ const OptionDescription: Record<string, string> = {
   saveloadinhash: "Persist text in URL hash",
   theme: "Light/Dark theme",
   readOnly: "Readonly",
+  preserveLocalState: "Preserve editor's local state"
```

Comment:
I think preserveLocalState is not descriptive enough as to what it's actually doing ( think I am likely to forget what that exactly means). How about: allowInputTextOverwrites?

Reply:
Also, could you add a description in the help README.md describing this in a bit more detail>